astropy/utils/introspection.py

```diff
@@ -86,7 +93,7 @@ def resolve_name(name, *additional_parts):
     return ret
 
 
-def minversion(module, version, inclusive=True):
+def minversion(module: ModuleType | str, version: str, inclusive: bool = True) -> bool:
```

Comment:
thanks a lot for doing this, I can never remember the return type of this function, I'm thrilled that soon my text editor will tell me !